notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    DOCKER_REPOSITORY = PROJECT_ID + "-repo-" + UUID
</pre><p>Simplify to:</p><p> </p><p>`DOCKER_REPOSITORY = f"{PROJECT_ID}-repo-unique"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>